services/movie_session.py

```diff
@@ -42,3 +42,18 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    movie_session = MovieSession.objects.get(
+        id=movie_session_id
+    )
+    result = []
+    for ticket in movie_session.tickets.all():
+        result.append(
+            {
+                "row": ticket.row,
+                "seat": ticket.seat
+            }
+        )
```

Comment:
There is no need to iterate through all ticket objects here, build your call from the `Ticket` model filtering by movie session id. You could obtain the required information using `values()` via ORM and convert the obtained info to a list in the return statement. This approach would be more efficient.